examples/machsuite/mach-common/Makefile

```diff
@@ -1,7 +1,9 @@
 
 BENCHMARKS=\
-	../mach-gemm_ncubed \
-	../mach-stencil2d
+	../mach-gemm_ncubed 
+#	../mach-stencil2d
```

Comment:
Are these meant to be commented out? If so, can you remove them in this pull request and add them as you add benchmarks?